cfme/tests/v2v/test_v2v_manual.py

```diff
@@ -0,0 +1,632 @@
+            2. Create migration plan with special chars in name
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. Successful migration from vmware to OSP
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_flavors_can_be_selected_creating_migration_plan():
+    """
+    title: OSP: Test flavors can be selected creating migration plan
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware67 to OSP
+            2. Create migration plan amd choose flavors
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. Successful migration from vmware to OSP with the selected flavor
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_creating_multiple_migration_plans_with_same_name():
+    """
+    title: OSP: Test duplicate names in creating migration plan
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        caseimportance: medium
+        caseposneg: negative
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create two migration plan with same name
+        expectedResults:
+            1.
+            2. Duplicate migration plan name should not be allowed.
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_vm_name_with_special_characters_can_be_imported():
+    """
+    title: OSP: Test if VM name with special characters can be imported (It
+    should allow such imports)
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        caseimportance: low
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan and import vm name containing special chars
+        expectedResults:
+            1.
+            2. Importing Vm with special chars in name should be allowed.
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_if_no_password_is_exposed_in_logs_during_migration():
+    """
+    OSP: Test if no password is exposed in logs during migration
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. logs should not show password during migration
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_associated_tags_before_and_after_migration_department_accounting_kind():
```

Comment:
RHV usecase automated already, same can be used for OSP too, remove this since we are fixing OSP failures next week -> `test_single_vm_migration_power_state_tags_retirement` .